src/table/components/head/ListBoxWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { stardust } from '@nebula.js/stardust';
+import { TableLayout } from '../../../types';
+
+export interface ListBoxWrapperRenderProps {
+  ref: React.MutableRefObject<HTMLElement | null>;
+  listboxInstance: stardust.FieldInstance | undefined;
+}
+
+interface ListBoxWrapperProps {
+  children: (props: ListBoxWrapperRenderProps) => JSX.Element;
+  layout: TableLayout;
+  embed: stardust.Embed | undefined;
+  columnIndex: number;
+}
+
+export const ListBoxWrapper = ({ children, embed, layout, columnIndex }: ListBoxWrapperProps) => {
+  const [listboxInstance, setListboxInstance] = useState<stardust.FieldInstance>();
+  const ref = useRef<HTMLElement>(null);
+
+  // embedding
+  useEffect(() => {
+    if (!layout || !embed) return;
+
+    embed.field(layout.qHyperCube.qDimensionInfo[columnIndex]?.qFallbackTitle).then((instance) => {
+      setListboxInstance(instance);
+    });
+  }, []);
+
+  // mounting
+  useEffect(() => {
+    if (!ref.current || !listboxInstance) return undefined;
+
+    listboxInstance.mount(ref.current, { title: ' ' });
```

Comment:
Why `title` with only a whitespace character?

Reply:
apparently, Listbox instance accepts an options while mounting but i've checked the `index.d.ts` files for it and there was an optional title(`title?: string`).

what we need here was that we did not want to show the title of columns there, and seems like either of passing `null` or `undefined` or even not passing title at all will still fallback into showing it anyway. So empty string would work in this case. Also i could not find any flag like `showTitle` or something like that to set it `false` in order to ignore the title.

Reply:
a comment it in the code for that would be nice

Reply:
Seems like a solid workaround to me, but as Max said, a comment for it would be nice